.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: csharp
+mono: none
+solution: EF-Adapter.sln
+dist: xenial
+dotnet: 
+    - 3.1
```

Comment:
it's .net core 3.1, this project is intended to support .net framework

Reply:
@ChangeTheCode

Reply:
I'm sorry. I'm not familiar with travis.yml. I take that from the EFCore-Adapter and did not adjust it. 
Can someone help me out here ?

Reply:
@Sagilio can you help on it?

Reply:
Travis does not support the .net framework well, we can use Github Actions or Azure Pipelines. Now, Casbin.NET and EFCore adapter project use the Github Actions mainly. 
You can delete the `.travis.yml` file directly.